package/Dockerfile

```diff
@@ -329,4 +329,12 @@ VOLUME /var/lib/kubelet
 VOLUME /var/lib/cni
 VOLUME /var/log
 
+LABEL "io.artifacthub.package.logo-url"="https://github.com/rancher/ui/blob/master/public/assets/images/logos/welcome-cow.svg" \
```

Comment:
Isn't it a problem that the url points to a dynamic reference that could be deleted?

Reply:
I made sure to use the same reference as the [charts does](https://github.com/rancher/rancher/blob/3ed01c34581afc6bcc4da77c78127d982e13a405/chart/Chart.yaml#L8), which is the same across all released charts I checked. Therefore, if we were to ever to create a `main` branch on that repository, we would have to keep the current `master` to avoid breaking this link to all released chart versions.

Reply:
Hi! I noticed that the logo URL points to a GitHub page (i.e., an HTML page where GitHub shows a rendering of the SVG).
I think this should point to the raw image URL instead (https://raw.githubusercontent.com/rancher/ui/master/public/assets/images/logos/welcome-cow.svg)